### PR TITLE
Add test for float kerning, make disabled

### PR DESCRIPTION
**Pull request by mitchhentges**
## Make automated test for #11482 

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes add a test for #11482

- [X] There are tests for these changes

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11483)

### CODE REVIEW

File: tests/wpt/mozilla/meta/MANIFEST.json [hunk 1]
```diff
@@ -1500,6 +1500,18 @@
...
+            "path": "css/float_kerning_a.html",
+            "references": [
+              [
+                "/_mozilla/css/float_kerning_ref.html",
+                "=="
+              ]
+            ],
+            "url": "/_mozilla/css/float_kerning_a.html"
+          }
+        ],
```
Comment (mitchhentges) [inline tests/wpt/mozilla/meta/MANIFEST.json]: There's a `.items.reftest` section and a `.reftest_nodes` section in `MANIFEST.json`. I'm not sure the pattern for where the test definition should go, so I put it in `.items.reftest`, and it's properly picked up by `./mach test-wpt`, so I think it's happy

**Comment by mitchhentges** [comment]
Am I not allowed to change the manifest?
```
$ bash etc/ci/manifest_changed.sh
diff --git a/tests/wpt/mozilla/meta/MANIFEST.json b/tests/wpt/mozilla/meta/MANIFEST.json
index c617a0f..42570f8 100644
--- a/tests/wpt/mozilla/meta/MANIFEST.json
+++ b/tests/wpt/mozilla/meta/MANIFEST.json
@@ -8516,6 +8516,18 @@
           "url": "/_mozilla/css/float_intrinsic_width_a.html"
         }
       ],
+      "css/float_kerning_a.html": [
+        {
+          "path": "css/float_kerning_a.html",
+          "references": [
+            [
+              "/_mozilla/css/float_kerning_ref.html",
+              "=="
+            ]
+          ],
+          "url": "/_mozilla/css/float_kerning_a.html"
+        }
+      ],
       "css/float_overflow_area_a.html": [
         {
           "path": "css/float_overflow_area_a.html",
The command "bash etc/ci/manifest_changed.sh" exited with 1.

**Comment by KiChjang** [comment]
Have you tried using `./mach create-wpt` to create tests?

**Comment by mitchhentges** [comment]
I re-did this according to `./mach create-wpt`, but `MANIFEST.json` is still changed. However, now _two_ places in the manifest have been updated. Maybe it will be happy now?

File: tests/wpt/mozilla/meta/MANIFEST.json (hunk 1 above)
Comment (mitchhentges) [inline tests/wpt/mozilla/meta/MANIFEST.json]: Yep, it needed to be added to both sections, now CI doesn't fail

**Comment by KiChjang** [comment]
Yep, Travis CI doesn't complain now.

@bors-servo try

**Comment by Ms2ger** [comment]
I'm not sure I see the point of this PR: you're adding a test, but not running it? Did you mean to mark it as failing instead?

**Comment by mitchhentges** [comment]
Ah, yes, I want to add it as failing. Still figuring out how the wpt test configuration happens

**Comment by Ms2ger** [comment]
See https://github.com/servo/servo/blob/e8e7c65/tests/wpt/metadata/html/rendering/non-replaced-elements/lists/li-type-supported.html.ini for an example

**Comment by Ms2ger** [comment]
Looks good in terms of metadata; r? @mbrubeck for the test

**Comment by mbrubeck** [comment]
Something is wrong; this test is not passing in Gecko or Blink. In both cases the horizontal position of the `.float` element in the test case is different than the position of the `.fixed` element in the reference. In Gecko the green/red/orange backgrounds are also showing; I think adding an explicit `line-height: 1;` fixes this.

I'm also not sure it will be possible to reproduce this bug using Ahem, because I don't think it has any kerning tables. :(  This may depend on something like #8374 so we can use a known font other than Ahem.

**Comment by mitchhentges** [comment]
Ah, yes, I think I got dinged by `./mach run` not using `Ahem` and falling back to some other font, and for some reason I thought that the other font was `Ahem`. Oops for the confusion, I'm not super sure how I can add a test for this now

**Pull request closed without merging**